src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,277 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Duke {
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+    public static final String LIST = "list";
+    public static final String DONE = "done";
+    public static final String EXIT = "bye";
+    public static final String DELETE = "delete";
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+    private static int numberOfTasks = 0;
+    private static File save;
+
+    public static void main(String[] args) {
+        saveCheck();
+        greet();
+        instructions();
+        chooseTask();
+    }
+
+    private static void saveCheck() {
```

Comment:
There are few or no comments generally. 
Including single-line method descriptors as javadocs would be helpful.